src/test/resources/features/login/Login.feature

```diff
@@ -10,7 +10,7 @@ Feature: Login with valid credentials & log out
 
     Examples:
       | username     | password     |
-      | standar_user | secret_sauce |
+      | standard_user | secret_sauce |
```

Comment:
Are you sure it is a good practice to use credentials directly in your feature file?